Lib/python/pyruntime.swg

```diff
@@ -30,6 +30,7 @@
 #else
 # include <Python.h>
 #endif
+#include "structmember.h"
```

Comment:
Seems to be a header from the Python API, so that should be: `#include <structmember.h>`

Reply:
Fixed.